common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -616,6 +617,14 @@ protected void addTracerTag(String key, String value) {
       return this;
     }
 
+    public Iterable<Map.Entry<String, String>> baggageItems() {
```

Comment:
PS: I'm not that excited about the OT-java change to make SpanBuilder inherit from SpanContext, but as long as that's part of the contract we may as well "do it right" and represent the parent context when we have one.